### PR TITLE
Standardise .md files to lowercase file name

### PR DESCRIPTION
**Pull request by rebekahlow-jy**
Changed non-lowercase .md file names to lowercase name convention.

### CODE REVIEW

File: book/se-pattern/introduction/apply.md [hunk 1]
```diff
@@ -1 +1 @@
-* :one: Do the exercise given in [SE-EDU: Addressbook Level 4: Apply Design Patterns](https://github.com/se-edu/addressbook-level4/blob/master/docs/LearningOutcomes.md#apply-design-patterns-lo-designpatterns)
+* :one: Do the exercise given in [SE-EDU: Addressbook Level 4: Apply Design Patterns](https://github.com/se-edu/addressbook-level4/blob/master/docs/Learningoutcomes.md#apply-design-patterns-lo-designpatterns)
```
Comment (damithc) [inline book/se-pattern/introduction/apply.md]: lower case `L`?

File: book/se-pattern/singleton/apply.md [hunk 2]
```diff
@@ -1 +1 @@
-* :one: Do the exercise given in [SE-EDU: Addressbook Level 4: Apply Design Patterns](https://github.com/se-edu/addressbook-level4/blob/master/docs/LearningOutcomes.md#apply-design-patterns-lo-designpatterns)
+* :one: Do the exercise given in [SE-EDU: Addressbook Level 4: Apply Design Patterns](https://github.com/se-edu/addressbook-level4/blob/master/docs/Learningoutcomes.md#apply-design-patterns-lo-designpatterns)
```
Comment (damithc) [inline book/se-pattern/singleton/apply.md]: Same here

**Review by damithc — CHANGES_REQUESTED** [review]
A quick look uncovered a possible oversight

**Comment by damithc** [comment]
Still not fixed. Should be `learningOutcomes.md` right? Also make sure the corresponding file is named correctly.

**Comment by rebekahlow-jy** [comment]
Sorry, it was an oversight on my part. It should have been LearningOutcomes.md as it was part of a URL. I've changed the parts and the URL link should be working now!

Update: It should not be learningOutcomes.md as the file is not contained in this repository, and can only be accessed by https://github.com/se-edu/addressbook-level4/blob/master/docs/LearningOutcomes.md#apply-design-patterns-lo-designpatterns

**Comment by damithc** [comment]
> It should not be learningOutcomes.md as the file is not contained in this repository, and can only be accessed by https://github.com/se-edu/addressbook-level4/blob/master/docs/LearningOutcomes.md#apply-design-patterns-lo-designpatterns

Sorry, my bad. I didn't notice it's a different repo.

**Comment by rebekahlow-jy** [comment]
It's alright, thanks for spotting the error! (:

**Review by damithc — APPROVED** [review]

**Comment by damithc** [comment]
@rebekahlow-jy In future, do each PR in a separate branch (i.e. not `master` branch).

**Comment by rebekahlow-jy** [comment]
Oops, will do that next time!

**Pull request merged**